### PR TITLE
[#11878] Add AccountRequest Rejection email generator.

### PR DESCRIPTION
**Pull request by EuniceSim142**
Part of #11878.

**Outline of Solution**
Add Email-generator methods for account request rejection email.

Example (with entire content passed in as rich text):
Screenshot 2024-04-06 at 4 30 17 PM

### CODE REVIEW

**Review by domoberzin — COMMENTED** [review]
Other than the lint checks failing, code looks good to me. Thanks for the work on this!

**Review by domoberzin — APPROVED** [review]
LGTM!

File: src/main/java/teammates/sqllogic/api/SqlEmailGenerator.java [hunk 1]
```diff
@@ -1029,6 +1029,18 @@ public EmailWrapper generateNewAccountRequestAcknowledgementEmail(AccountRequest
         return email;
     }
 
+    /**
+     * Generates the email to be sent to instructor when their account request has been rejected by admin.
+     */
+    public EmailWrapper generateAccountRequestRejectionEmail(AccountRequest accountRequest, String title, String content) {
```
Comment (xenosf) [inline src/main/java/teammates/sqllogic/api/SqlEmailGenerator.java:1035]: I think prof mentioned he wanted the email to be BCC-ed to the support email so the admin gets a copy as well

**Review by xenosf — CHANGES_REQUESTED** [review]
Other than needing to BCC the support email, LGTM!

**Review by xenosf — APPROVED** [review]
LGTM. Thanks for the work!

**Review by domoberzin — APPROVED** [review]
LGTM

**Pull request merged**